module/VuFind/src/VuFind/ILS/Driver/SierraRest.php

```diff
@@ -837,6 +859,20 @@ public function getMyTransactions($patron, $params = [])
             }
             $transactions[] = $transaction;
         }
+        if ($this->config['InnReach']['enabled'] ?? false) {
+            $n = 0;
+            foreach ($transactions as $transaction) {
+                $irIdentifier = $this->config['InnReach']['identifier'];
+                if ($transaction['item_id'] && strstr($transaction['item_id'], $irIdentifier)) {
+                    $irCheckoutId = $transaction['checkout_id'];
+                    $irItemId = $transaction['item_id'];
+                    $innReach = $this->getInnReachCheckoutTitleInfoFromId($irCheckoutId, $irItemId);
```

Comment:
As far as I can see, getInnReachCheckoutTitleInfoFromId return value should be checked before accessing the array elements.